lowlevel/src/motor-board/motion/MotionController.cpp

```diff
@@ -355,103 +128,76 @@ int32_t MotionController::get_data_from_pid(PID_FP &pid, uint8_t requested_data)
       break;
     case PID_DATA_ID::DERIVATIVE:ret_data = pid.get_derivative();
       break;
-    case PID_DATA_ID::ERROR:ret_data = pid.get_error();
+    case PID_DATA_ID::DELTA:ret_data = pid.get_error();
       break;
     default:ret_data = -1;
       break;
   }
   return ret_data;
 }
 
-/**
- * Gets the PID data associated with the PID_DATA_ID requested_data, for a given PID_ID id
- */
+
 int32_t MotionController::get_pid_data(uint8_t id, uint8_t requested_data) {
   int32_t ret_val;
   switch (id) {
     case PID_ID::PID_LEFT_SPEED:ret_val = get_data_from_pid(pid_speed_left, requested_data);
       break;
     case PID_ID::PID_RIGHT_SPEED:ret_val = get_data_from_pid(pid_speed_right, requested_data);
       break;
-    case PID_ID::PID_TRANSLATION:ret_val = get_data_from_pid(pid_translation, requested_data);
-      break;
-    case PID_ID::PID_ROTATION:ret_val = get_data_from_pid(pid_rotation, requested_data);
-      break;
     default:ret_val = -1;
       break;
   }
   return ret_val;
 }
 
-void MotionController::set_raw_pwm(Motor::Side side, int16_t pwm) {
-  switch (side) {
-    case Motor::Side::LEFT :motor_left.set_pwm(pwm);
-      break;
-    case Motor::Side::RIGHT:motor_right.set_pwm(pwm);
-      break;
-  }
+
+void MotionController::set_raw_pwm(int16_t left, int16_t right) {
+  motor_left.set_pwm(left);
+  motor_right.set_pwm(right);
 }
 
+
 void MotionController::set_kp(uint8_t id, uint32_t k) {
```

Comment:
Use 2 parameters instead of a switch case

Reply:
Done in 8034602

Reply:
Done in 8034602